web-app/src/lib/db/services/job.service.ts

```diff
@@ -151,42 +147,35 @@ export async function startJob(
         },
       },
     });
-    await prisma.creditTransaction.update({
-      where: {
-        id: creditTransaction.id,
-      },
-      data: {
-        status: "SUCCEEDED",
-      },
-    });
-
     return job;
   } catch (error) {
-    const job = await prisma.creditTransaction.update({
-      where: {
-        id: creditTransaction.id,
-      },
-      data: {
-        status: "FAILED",
-        errorNote: error instanceof Error ? error.message : "Unknown error",
-        errorNoteKey: "Job.CreationFailed",
-      },
-      select: {
-        job: true,
-      },
-    });
-    if (job && job.job) {
-      await prisma.job.update({
-        where: {
-          id: job.job.id,
-        },
-        data: {
-          status: "FAILED",
-          errorNote: error instanceof Error ? error.message : "Unknown error",
-          errorNoteKey: "Job.CreationFailed",
-        },
-      });
-    }
+    // TODO: update credit transaction status to failed
```

Comment:
should this commented out code be in the PR?

Reply:
No, we can either delete the CreditTransaction or create a new one to refund the original amount. For now, I have implemented a deletion since it occurs right at the beginning of the start_job process. f0db7f953f131582631a9182c68c8022763b2fca